packages/noya-state/src/exportOptions.ts

```diff
@@ -0,0 +1,38 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+
+type ExportScale = {
+  scale: number;
+  absoluteSize: number;
+  visibleScaleType: Sketch.VisibleScaleType;
+};
+
+export function parseScale(
+  scaleText: string,
+  rect: Sketch.Rect,
```

Comment:
I didn't really realize before, but if we include the rect here, then we're going to have to pass it all the way in to our input validation, and it really isn't necessary there. I think we're better off parsing & validating just the user input `[size, visibleScaleType]` here, and handling the scale/rect calculation in the reducer